yellowbrick/features/projection.py

```diff
@@ -0,0 +1,269 @@
+
+    classes: list, default: None
+        a list of class names for the legend
+        If classes is None and a y value is passed to fit then the classes
+        are selected from the target vector.
+
+    color : list or tuple of colors, default: None
+        Specify the colors for each individual class.
+
+    colormap : string or cmap, default: None
+        Optional string or matplotlib cmap to colorize points.
+        Use either color to colorize the points on a per class basis or
+        colormap to color them on a continuous scale.
+
+    target_type : str, default: "auto"
+        Specify the type of target as either "discrete" (classes) or "continuous"
+        (real numbers, usually for regression). If "auto", then it will
+        attempt to determine the type by counting the number of unique values.
+
+    projection : int, default: 2
+        Dimension of the Projection Visualizer.
+
+    alpha : float, default: 0.75
+        Specify a transparency where 1 is completely opaque and 0 is completely
+        transparent. This property makes densely clustered points more visible.
+
+    kwargs : dict
+        Keyword arguments that are passed to the base class and may influence
+        the visualization as defined in other Visualizers.
+    """
+    
+    def __init__(self, ax=None, features=None, classes=None, color=None,
+             colormap=None, target_type="auto", projection=2, alpha=0.75, **kwargs):
+        super(ProjectionVisualizer, self).__init__(ax=ax, features=features, 
+                                                     classes=classes, color=color,
+                                                     colormap=colormap, 
+                                                     target_type=target_type, **kwargs)
+
+        if projection not in frozenset((2, 3, '2D', '3D')):
+            raise YellowbrickValueError("Projection dimensions must be either 2 or 3")
+
+        self.projection = projection
+        self.alpha = alpha
+        self._cax = None
+
+    @property
+    def cax(self):
+        """
+        The axes of the colorbar, right of the scatterplot.
+        """
+        if self._cax is None:
+            raise AttributeError(
+                "This visualizer does not have an axes for colorbar"
+            )
+        return self._cax       
+        
+    def _layout(self):
+        """
+        Creates the layout for colorbar when target type is continuous.
+        The colorbar is added to the right of the scatterplot.
+        """
+        # Ensure matplotlib version compatibility
+        if make_axes_locatable is None:
+            raise YellowbrickValueError((
+                "heatmap requires matplotlib 2.0.2 or greater "
+                "please upgrade matplotlib or set heatmap=False on the visualizer"
+            ))
+
+        # Create the new axes for the colorbar
+        divider = make_axes_locatable(self.ax)
+        self._cax = divider.append_axes("right", size="5%", pad=0.3)
```

Comment:
I'm sure you're still working on it - but just wanted to make a note so we didn't miss it - the `cax` is only required if `_target_type == CONTINUOUS` and we also want to make sure that no `_cax` is already there (e.g. from a different layout call).

Reply:
I've been thinking about this, and I think we should make it `layout()` instead of `_layout()` so that users can organize the axes as required.